grails-app/services/org/bbop/apollo/FeatureService.groovy

```diff
@@ -1642,22 +1644,23 @@ public void setTranslationEnd(Transcript transcript, int translationEnd) {
                 JSONObject jsonProperty = new JSONObject();
                 JSONObject jsonPropertyType = new JSONObject()
                 if (property instanceof Comment) {
-                    //  TODO: This is a hack
-                    jsonPropertyType.put(FeatureStringEnum.NAME.value, "comment")
                     JSONObject jsonPropertyTypeCv = new JSONObject()
                     jsonPropertyTypeCv.put(FeatureStringEnum.NAME.value, FeatureStringEnum.FEATURE_PROPERTY.value)
                     jsonPropertyType.put(FeatureStringEnum.CV.value, jsonPropertyTypeCv)
+
                     jsonProperty.put(FeatureStringEnum.TYPE.value, jsonPropertyType);
+                    jsonProperty.put(FeatureStringEnum.NAME.value, FeatureStringEnum.COMMENT.value);
                     jsonProperty.put(FeatureStringEnum.VALUE.value, property.getValue());
                     properties.put(jsonProperty);
                     continue
                 }
                 if (property.tag == "justification") {
-                    jsonPropertyType.put(FeatureStringEnum.NAME.value, "justification")
                     JSONObject jsonPropertyTypeCv = new JSONObject()
                     jsonPropertyTypeCv.put(FeatureStringEnum.NAME.value, FeatureStringEnum.FEATURE_PROPERTY.value)
                     jsonPropertyType.put(FeatureStringEnum.CV.value, jsonPropertyTypeCv)
+
                     jsonProperty.put(FeatureStringEnum.TYPE.value, jsonPropertyType);
+                    jsonProperty.put(FeatureStringEnum.NAME.value, "justification");
```

Comment:
probably good to have "justification" move be a varianble in FeatureStringEnum . . but I wouldn't hold this up for that.